OnBaseDocsApi/log4net.config

```diff
@@ -0,0 +1,24 @@
+﻿<log4net>
+    <root>
+        <level value="Debug" />
+        <appender-ref ref="file" />
+        <!-- <appender-ref ref="console" /> -->
+    </root>
+    <appender name="console" type="log4net.Appender.ConsoleAppender">
+        <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="%date %level %logger - %message%newline" />
+        </layout>
+    </appender>
+    <appender name="file" type="log4net.Appender.RollingFileAppender">
+        <file value="./logs/" />
+        <appendToFile value="true" />
+        <rollingStyle value="Composite" />
+        <maxSizeRollBackups value="10" />
+        <datePattern value="yyyy-MM-dd'.log'" />
+        <maximumFileSize value="10MB" />
+        <staticLogFileName value="false" />
+        <layout type="log4net.Layout.PatternLayout">
+            <conversionPattern value="%date [%thread] %level %logger - %message%newline" />
+        </layout>
+    </appender>
```

Comment:
This file should not be hardcoded, instead, it should be configurable by our config file.

Reply:
I don't believe there are any variables we can use for this config file. There is an option, I believe, if we use a different appender to use environment variables, but not loading variables from another config file.

Reply:
Aren't you able to use template file to generate this file? I believe it's a very common way to render a file from a config file or with environment variables.

Reply:
This config file need to be there at launch time. The AssemblyInfo has a reference to this file and this file get loaded before any API code is executed.

Reply:
If you think it's impossible, at the very least you should move this file to config repo instead of hardcoded here. In this case, you will have to put all onbase-docs v1 config files into its folder and have git pull this file from that repo.